include/battle.h

```diff
@@ -675,6 +675,24 @@ struct BattleStruct
         typeArg = gBattleMoves[move].type;                            \
 }
 
+#define GET_MOVE_CALLED(move, targetArg)                              \
```

Comment:
Why make this a macro instead of a function? Not saying it's wrong, I'm just curious.